lib/remote/apilistener.hpp

```diff
@@ -85,7 +85,7 @@ class ApiListener final : public ObjectImpl<ApiListener>
 
 	/* filesync */
 	static Value ConfigUpdateHandler(const MessageOrigin::Ptr& origin, const Dictionary::Ptr& params);
-	static void HandleConfigUpdate(const MessageOrigin::Ptr& origin, const Dictionary::Ptr& params);
+	void HandleConfigUpdate(const MessageOrigin::Ptr& origin, const Dictionary::Ptr& params);
```

Comment:
This breaks the call in https://github.com/Icinga/icinga2/blob/3d14b78432570476115d49fcf26e8210d6aa15fd/lib/remote/apilistener-filesync.cpp#L317